src/main/java/org/testng/xml/internal/XmlSuiteUtils.java

```diff
@@ -7,47 +7,36 @@
 import org.testng.xml.XmlSuite;
 import org.testng.xml.XmlTest;
 
+import java.util.Iterator;
 import java.util.List;
 import java.util.Set;
 
 /**
  * A utility class that exposes helper methods to work with {@link XmlSuite}
  */
 public final class XmlSuiteUtils {
-
-    private XmlSuiteUtils() {
-        //Utility class. Defeat instantiation.
-    }
+    private List<XmlSuite> cloneSuites = Lists.newArrayList();
```

Comment:
Create a new class if you have specific need, or rename this one with its comment

Reply:
> rename this one with its comment
@krmahadevan and @juherr I prefer to keeping all the method still in this class, and changing class javadoc simply as follow:
A utility class to work with {@link XmlSuite}

What do you think?

Reply:
A utility class has no instance and only static methods.

If you have instances, you need to find an appropriate name.